inst/resources/gitbook/js/plugin-bookdown.js

```diff
@@ -97,6 +97,8 @@ gitbook.require(["gitbook", "lodash", "jQuery"], function(gitbook, _, $) {
     // highlight the current section in TOC
     var href = window.location.pathname;
     href = href.substr(href.lastIndexOf('/') + 1);
+    // accentuated characters need to be decoded (#819)
+    href = decodeURI(href);
```

Comment:
I guess you should call `decodeURIComponent()` here.

Reply:
I hesitated between both and it seemed special characters URI separators (; / ? : @ & = + $ , #) seems to already be handled. I tested with a title `litérature ?` and `?` was not put inside the filename, so not encoded. 

However, it also will work as expected with `decodeURIComponent` so it is ok to use it instead.